crates/typst/src/image.rs

```diff
@@ -342,23 +342,50 @@ fn load_svg_fonts(
         name
     };
 
-    // Load fallback family.
-    let mut fallback_usvg_compatible = None;
-    if let Some(family) = loader.fallback_family() {
-        fallback_usvg_compatible = load(family);
-    }
-
-    // Find out which font families are referenced by the SVG.
+    // Determine the best font for each text node.
     traverse_svg(&tree.root, &mut |node| {
         let usvg::NodeKind::Text(text) = &mut *node.borrow_mut() else { return };
         for chunk in &mut text.chunks {
             for span in &mut chunk.spans {
-                for family in &mut span.font.families {
-                    if family.is_empty() || load(family).is_none() {
-                        if let Some(fallback) = &fallback_usvg_compatible {
-                            *family = fallback.into();
+                let text = chunk
+                    .text
+                    .chars()
+                    .skip(span.start)
+                    .take(span.end - span.start)
+                    .collect::<EcoString>();
```

Comment:
As far as I can see from usvg's docs, the offsets are in bytes, not chars so this should perhaps be just 

```rust
let Some(text) = chunk.text.get(span.start..span.end) else { continue };
```

Reply:
> A span start in UTF-8 codepoints.

Maybe I misunderstand, but doesn't that mean chars?

Reply:
Apparently we were reading different docs! Your snippet is from usvg 0.32, while I read 0.35 which states:

> A span start in bytes.

So with that said, your correct was correct initially, but since you opened the PR, usvg was bumped to 0.35, so it's still bytes in the end.